browser/ui/views/text_recognition_dialog_view.cc

```diff
@@ -90,12 +95,21 @@ void TextRecognitionDialogView::StartExtractingText(const SkBitmap& image) {
   header_label_->SetText(brave_l10n::GetLocalizedResourceUTF16String(
       IDS_TEXT_RECOGNITION_DIALOG_HEADER_IN_PROGRESS));
 
+#if BUILDFLAG(IS_MAC)
   base::ThreadPool::PostTaskAndReplyWithResult(
       FROM_HERE,
       {base::MayBlock(), base::TaskShutdownBehavior::SKIP_ON_SHUTDOWN},
       base::BindOnce(&text_recognition::GetTextFromImage, image),
       base::BindOnce(&TextRecognitionDialogView::OnGetTextFromImage,
                      weak_factory_.GetWeakPtr()));
+#endif
+
+#if BUILDFLAG(IS_WIN)
+  helper_ = std::make_unique<text_recognition::TextRecognitionHelperWin>();
+  helper_->GetTextFromImage(
```

Comment:
maybe we should do extraction and COM instances initialization on a separate thread?

Reply:
Done. `TextRecognitionHelperWin` post those tasks to another thread.